docs/pages/inboxes/manage-inboxes.mdx

```diff
@@ -0,0 +1,189 @@
+# Manage XMTP inboxes
+
+The first time someone uses an installation of your app, your app creates an inbox ID and installation ID associated with their wallet address. To do this, you [create a client](https://docs.xmtp.org/inboxes/build-inbox#create-an-xmtp-client) for their wallet address. 
+
+The client creates an inbox ID and installation ID associated with the wallet address. By default, this wallet address is designated as the recovery address. A recovery address will always have the same inbox ID and cannot be reassigned to a different inbox ID. 
+
+<div>
+  <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/0fa5d2dd3c00fb25c0adb7a615408574eb6864a7/docs/pages/img/create-inbox-install.png" width="250px" />
+</div>
+
+Each time you make a subsequent call to create a client for the same wallet address, the client uses the same inbox ID, but creates a new installation ID.
```

Comment:
Each time you make a subsquent call to create client when a local database is not present.

In theory you could call create 100 times and it will fetch from the local database if it is present not creating a new installation unless it is infact a new installation.